src/db/Procedures.ts

```diff
@@ -36,13 +36,7 @@ export default class Procedures {
       user: 'kpmg',
       database,
     });
-    client.connect((err: any) => {
-      if (err) {
-        console.error('connection error', err.stack);
-      } else {
-        console.log('connected');
-      }
-    });
```

Comment:
Why do we not need these error messages?

Reply:
We should be doing this with a custom logger rather than just printing to console (creates linting error)